nengo_loihi/simulator.py

```diff
@@ -481,8 +481,8 @@ def handle_host2chip_communications(self):  # noqa: C901
                         for j, s in enumerate(spike_input.spikes[sent_count]):
                             if s:
                                 for output_axon in spike_input.axon_ids:
-                                    items.append((
-                                        sent_count, *output_axon[j]))
+                                    items.append(
+                                        (sent_count,) + output_axon[j])
```

Comment:
Is `output_axon[j]` a tuple? Does it get automatically converted to a tuple because the left hand side is a tuple? My instinct is to ensure the right hand side is also a tuple, but if Python does that automatically then it's no big deal.

Reply:
So Python will not automatically convert (adding a list and a tuple results in an error, when done in either order, at least in Python 3). I'm pretty sure that `output_axon` is a tuple. (I recall testing this and having it run on Loihi, but somebody else should double-check.)